scripts/close-issue-channel/common.js

```diff
@@ -1,16 +1,26 @@
-export const closeIssueChannel = (client, issue) => {
+export const closeIssueChannel = async (client, issue) => {
 	let success = false;
 	try {
 		await client.guilds.fetch();
 		const { channels } = client.guilds.cache.get(process.env.GUILD_ID);
-		const hubChannel = channels.cache.find((c) => c.position === 0);
-		const closedChannel = channels.cache.find((c) => c.name === `website-issue-${issue.number}`);
-		// TODO: Check if hub channel is always at position 0.
-		console.log({ hubChannel, closedChannel, channels: [...channels.cache] });
-		// TODO: Fetch all messages from `closedChannel` text channel.
-		// TODO: Print messages out to TXT file format.
-		// TODO: Send TXT file to `hubChannel` text channel.
-		// TODO: Delete `closedChannel` text channel.
+		const hubChannel = channels.cache.find(
+			(c) => c.parentId === process.env.HUB_ID && c.position === 0

```

Comment:
The `HUB_ID` should be matching the parent category? So the code should've been
`c.id == process.env.HUB_ID`, unless the `hubChannel` variable is pointing to a
channel inside the category, but that's nto a good idea.